src/tool.rs

```diff
@@ -395,12 +400,19 @@ impl Tool {
         flags
     }
 
+    /// The family of this tool, representing convention of arguments etc.
+    pub fn family(&self) -> ToolFamily {
+        self.family
+    }
+
     /// Whether the tool is GNU Compiler Collection-like.
+    #[deprecated = "Consider matching against the ToolFamily returned by family() instead"]
```

Comment:
Not sure if we should deprecate this now cc @madsmtm what do you think?

Reply:
Wasn't sure either, might be a bit harsh but otherwise we could forget about it.

Reply:
At the same time matching against `self.family()` for single variants is a bit ugly as you've seen.